pandas/tests/test_graphics.py

```diff
@@ -1329,10 +1329,6 @@ def test_plot(self):
                                      subplots=True, use_index=False)
         self._check_axes_shape(axes, axes_num=4, layout=(4, 1))
 
-        df = DataFrame({'x': [1, 2], 'y': [3, 4]})
```

Comment:
and add another test with only takes >=1.5.2 and tests for the correct exception.